spec/dummy/spec/overlap_validator_spec.rb

```diff
@@ -19,6 +20,30 @@
       subject.validate(meeting)
       meeting.errors[:optional_key].should eq ["Message content"]
     end
+  end
+
+  context "Updates booking" do
+    context "A model with a integer as a primary key" do
+      it "Updates the relevant record" do
+        meeting = Meeting.first
+        meeting.update(starts_at: '2012-01-05'.to_date, ends_at: '2012-02-05'.to_date)
+
+        expect(meeting.valid?).to eq true
+      end
+    end
+
+    before(:all) do
+      SecureMeeting.delete_all
+    end
+
+    context "A model with a UUID as a primary key" do
+      it "Updates the relevant record" do
+        meeting_uuid = FactoryGirl.create(:secure_meeting)
+        meeting_uuid.update(starts_at: '2012-01-05'.to_date, ends_at: '2012-02-05'.to_date)
+
+        expect(meeting_uuid.valid?).to eq true
+      end
+    end
 
```

Comment:
I have a feeling the maintainers might ask you to move this into a model test as something like spec/models/secure_meeting_spec.rb. 

Most of the tests seem to go something like:
```ruby
meeting = FactoryGirl.build #etc etc
meeting.should be_valid # or .should_not be_valid
meeting.errors[:starts_at].should be_empty # or .should_not be_empty
meeting.errors[:ends_at].should be_empty
```